protostar/self/protostar_directory_test.py

```diff
@@ -20,11 +20,3 @@ def test_directory_root_path(script_root: Path):
     protostar_directory = ProtostarDirectory(script_root)
 
     assert protostar_directory.directory_root_path == script_root.parent.parent
-
-
-def test_test_only_cairo_path(script_root: Path):
-    protostar_directory = ProtostarDirectory(script_root)
-
-    cairo_path = protostar_directory.protostar_test_only_cairo_packages_path
-
-    assert Path(script_root / "cairo") == cairo_path
```

Comment:
Why'd you removed this test at the same time adding one function in a tested file? Logically, you should add a test here.

Reply:
because this test is really really unnecessary. 

1. it tests the implementation, not the functionality 
3. makes it harder to maintain the code, not easier (what majority of tests should do)